### PR TITLE
Feat_schemas_import

### PR DESCRIPTION
**Pull request by SpliiT**
Import schemas

### CODE REVIEW

File: composables/api_fetch.js [hunk 1]
```diff
@@ -5,14 +5,16 @@ export function api_fetch(
   { schema, params },
   { request_error_function, response_function, response_error_function } = {},
 ) {
+  // console.log("api_fetch", schema.route, schema.methods)
```
Comment (JulienChampagnol) [inline composables/api_fetch.js:8]: Remove console.logs

File: components/FileSelector.vue [hunk 2]
```diff
@@ -2,13 +2,17 @@
...
   />
 </template>
 
 <script setup>
-  import schema from "@/assets/schemas/FileSelector.json"
+  import schemas from "@geode/opengeodeweb-back/schemas.json"
+
+  const schema = schemas.opengeodeweb_back.allowed_files
+
+  const json_data = schemas
```
Comment (JulienChampagnol) [inline components/FileSelector.vue:15]: What is this used for?

File: components/ExtensionSelector.vue [hunk 3]
```diff
@@ -59,8 +59,9 @@
 </template>
 
 <script setup>
-  import schema from "@/assets/schemas/ExtensionSelector.json"
-
+  import schemas from "@geode/opengeodeweb-back/schemas.json"
+  const schema = schemas.opengeodeweb_back.geode_objects_and_output_extensions
+  const json_data = schemas
```
Comment (JulienChampagnol) [inline components/ExtensionSelector.vue:64]: Same here

File: components/ObjectSelector.vue [hunk 4]
```diff
@@ -50,7 +50,9 @@
 
 <script setup>
   import geode_objects from "@/assets/geode_objects"
-  import schema from "@/assets/schemas/ObjectSelector.json"
+  import schemas from "@geode/opengeodeweb-back/schemas.json"
+
+  const schema = schemas["opengeodeweb_back"]["allowed_objects"]
```
Comment (JulienChampagnol) [inline components/ObjectSelector.vue:55]:
```suggestion
  const schema = schemas.opengeodeweb_back.allowed_objects
```

File: composables/api_fetch.js [hunk 5]
```diff
@@ -25,8 +27,23 @@ export function api_fetch(
     throw new Error(schema.$id.concat(": ", ajv.errorsText()))
   }
   geode_store.start_request()
+  var methods = []
+  for (const method of schema.methods) {
```
Comment (JulienChampagnol) [inline composables/api_fetch.js:31]: To improve, filter

**Review by JulienChampagnol — APPROVED** [review]

**Comment by JulienChampagnol** [comment]
Think about the unit test for api_fetch and multiple calls with the same schema

File: components/CrsSelector.vue [hunk 6]
```diff
@@ -64,6 +66,7 @@
   async function get_crs_table() {
     const params = { input_geode_object }
     toggle_loading()
+    console.log("schema", schema.methods)
```
Comment (BotellaA) [inline components/CrsSelector.vue:69]: clean all these consoles

File: package.json [hunk 7]
```diff
@@ -44,7 +45,10 @@
     "@vueuse/core": "^10.7.2",
     "@vueuse/nuxt": "^10.7.2",
     "ajv": "^8.12.0",
+    "fs": "^0.0.1-security",
+    "glob": "^10.3.10",
     "lodash": "^4.17.21",
+    "path": "^0.12.7",
```
Comment (BotellaA) [inline package.json:51]: why do you need these?

Comment (SpliiT) [reply package.json:51]: I think this showed up when I did npm i geode, should I delete them?

Comment (BotellaA) [reply package.json:51]: try to delete them

**Review by BotellaA — APPROVED** [review]

**Pull request merged**